.buildkite/test.cmd

```diff
@@ -0,0 +1,9 @@
+@echo "Building using bash and build.sh"
+"C:\Program Files\git\bin\bash" .buildkite/test.sh
```

Comment:
Building on windows requires exactly that git, nothing else. Yup. It's specified in the testbot docs, etc. Yes, fragile otherwise.  But we can't really do anything without a predictable particular bash.